### PR TITLE
Add test for moving files to data disk

### PR DESCRIPTION
**Pull request by frmunozp**
This PR is part of the LISA ramp-up. The objective was to write a LISA test where a file is moved onto a disk.

### CODE REVIEW

File: microsoft/testsuites/core/disk_move.py [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+        priority=4,
+        requirement=simple_requirement(
+            disk=schema.DiskOptionSettings(
+                disk_type=schema.DiskType.StandardHDDLRS,
+                data_disk_count=search_space.IntRange(min=1),
+            )
+        ),
+    )
+    def check_disk_move_operation_frmunozp(self, node: Node, log: Logger) -> None:
+        test_file = "~/test.txt"
```
Comment (dsrivastavv) [inline microsoft/testsuites/core/disk_move.py]: These can be a constant on the class-level

File: microsoft/testsuites/core/disk_move.py [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+        assert_that(output).matches(test_str)
+
+        data_disks = node.features[Disk].get_raw_data_disks()
+        mount_point = "demo"
+
+        node.tools[Mount].mount(data_disks[0], mount_point, format=True)
+
+        if node.shell.exists(node.get_pure_path(mount_point)):
+            node.execute(
+                f"mv {test_file} {mount_point}",
```
Comment (dsrivastavv) [inline microsoft/testsuites/core/disk_move.py]: We try not to run commands directly and instead create a tool for it. This is more pythonic way and cleaner.

File: microsoft/testsuites/core/disk_move.py [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+                shell=True,
+                sudo=True,
+                expected_exit_code=0,
+                expected_exit_code_failure_message=(
+                    f"failed to move {test_file} to disk"
+                ),
+            )
+
+        output = cat.read(f"{mount_point}/test.txt")
+        assert_that(output).is_equal_to(test_str)
```
Comment (dsrivastavv) [inline microsoft/testsuites/core/disk_move.py]: We should always add a message to justify why this is expected program behavior

**Review by dsrivastavv — COMMENTED** [review]
Nice work! I have added few comments to improve code quality

File: microsoft/testsuites/core/disk_move.py (hunk 1 above)
Comment (frmunozp) [reply microsoft/testsuites/core/disk_move.py]: I moved the variables to the class level.

**Comment by squirrelsc** [comment]
@frmunozp please merge the commits, to make sure each commit is meaningful. We organize commits by its content, not CR progresses.

File: microsoft/testsuites/core/disk_move.py [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+
+@TestSuiteMetadata(
+    area="disk storage",
+    category="functional",
+    description="""
+    Test suite exercise for ramp-up
+    """,
+)
+class DiskStorage(TestSuite):
+    test_file = "~/test.txt"
```
Comment (dsrivastavv) [inline microsoft/testsuites/core/disk_move.py:28]: This should follow CAPITALIZED_SNAKE_CASE format: TEST_FILE

File: microsoft/testsuites/core/disk_move.py [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+        priority=4,
+        requirement=simple_requirement(
+            disk=schema.DiskOptionSettings(
+                disk_type=schema.DiskType.StandardHDDLRS,
+                data_disk_count=search_space.IntRange(min=1),
+            )
+        ),
+    )
+    def check_disk_move_operation_frmunozp(self, node: Node, log: Logger) -> None:
+
```
Comment (dsrivastavv) [inline microsoft/testsuites/core/disk_move.py:51]: I tend to break tests into three clear sections:

1. Setup
2. Running test
3. checking

Sometime 2 and 3 might be intertwined, but it's usually better to have this separation with comments if possible.

**Comment by dsrivastavv** [comment]
@frmunozp Any updates on this?

**Comment by dsrivastavv** [comment]
Nice work! Closing this task

**Pull request closed without merging**